packages/components/organisms/f-header/src/components/Header.vue

```diff
@@ -248,7 +254,16 @@ html:global(.is-navInView) {
 
     .c-header--highlightBg {
         background-color: $color-support-brand-01;
-        min-height: 88px;
+
+        @include media('>mid') {
+            min-height: 88px;
```

Comment:
Does this rule still need to be there now with the new prop? (should the header just be the same height, even when highlighted?)

Reply:
Yep, the highlight theme header is currently taller than the default themed header.